kafka-rest/src/test/java/io/confluent/kafkarest/unit/ConsumerManagerTest.java

```diff
@@ -364,6 +364,119 @@ public void testMultipleTopicSubscriptionsFail() throws Exception {
     EasyMock.verify(mdObserver, consumerFactory);
   }
 
+
+  @Test
+  public void testBackoffMsControlsPollCalls() throws Exception {
+    Properties props = new Properties();
+    props.setProperty(KafkaRestConfig.CONSUMER_ITERATOR_BACKOFF_MS_CONFIG, "250");
+    // This setting supports the testConsumerOverrides test. It is otherwise benign and should
+    // not affect other tests.
+    props.setProperty("exclude.internal.topics", "false");
+    config = new KafkaRestConfig(props, new SystemTime());
+    mdObserver = EasyMock.createMock(MetadataObserver.class);
+    consumerFactory = EasyMock.createMock(ConsumerManager.ConsumerFactory.class);
+    consumerManager = new ConsumerManager(config, mdObserver, consumerFactory);
+    expectCreateNoData();
+    EasyMock.expect(mdObserver.topicExists(topicName)).andReturn(true);
+    EasyMock.replay(mdObserver, consumerFactory);
+    String cid = consumerManager.createConsumer(groupName,
+            new ConsumerInstanceConfig(EmbeddedFormat.BINARY));
+    Future f = readTopicFuture(cid, topicName, Long.MAX_VALUE, new ConsumerReadCallback<byte[], byte[]>() {
+      @Override
+      public void onCompletion(List<? extends ConsumerRecord<byte[], byte[]>> records,
+                               RestException e) {
+        sawCallback = true;
```

Comment:
Unless I'm missing something, we don't appear to do any validation of these fields.

Reply:
There are a few more with the same problem. Maybe we could just create a function which does the following:
```java
f.get();
assertTrue("Callback not called", sawCallback);
assertNull("Callback exception should not be populated", actualException);
```
Then we can call it at the end of each function instead of `f.get()`.